Sources/LinguaLib/Infrastructure/SwiftLocalizeGenerator/Generator/LocalizedSwiftCodeGenerator.swift

```diff
@@ -46,6 +46,10 @@ private extension LocalizedSwiftCodeGenerator {
 
 private extension String {
   func commented() -> String {
-    "/// " + replacingOccurrences(of: "\n", with: "\\n")
+    let escapedString = self
+      .replacingOccurrences(of: "\n", with: "\\n")
+      .replacingOccurrences(of: "\r", with: "\\r")
```

Comment:
Maybe it would make sense to handle all common escape sequences here?

Additional:
```
\t - Tab
\\" - Double quote
\\' - Single quote
\\ - Backslash
\0 - Null character
```

Or, not sure what the logic here is, but we can also ascii escape the whole string with system unicode scalars:

```swift
extension Unicode.Scalar {
  var asciiEscaped: String { escaped(asASCII: true) }
}

extension StringProtocol {
  var asciiEscaped: String {
    unicodeScalars.map(\.asciiEscaped).joined()
  }
}
```

Reply:
@dejanskledar no need since these will be as a commented out which serves as a translation usage helper. The only issues were about new lines. Thanks, let me know if it makes sense

Reply:
I see, makes sense then!